pkg/archive/archive.go

```diff
@@ -633,6 +647,11 @@ func TarWithOptions(srcPath string, options *TarOptions) (io.ReadCloser, error)
 				}
 
 				if err := ta.addTarFile(filePath, relFilePath); err != nil {
+					if err == errSkipFilePermissions {
+						// GH18120. Note next line is deliberately NOT logrus or %q
+						fmt.Printf("Skipping '%s' - Permission was denied\n", filePath)
```

Comment:
Shouldn't "skipping" files be a fatal error? i.e. As a user, I told Docker to build an image from the specified context, but Docker wasn't able to complete that?